src/autocomplete/UserProvider.tsx

```diff
@@ -56,7 +56,7 @@ export default class UserProvider extends AutocompleteProvider {
         this.matcher = new QueryMatcher([], {
             keys: ['name'],
             funcs: [obj => obj.userId.slice(1)], // index by user id minus the leading '@'
-            shouldMatchPrefix: true,
+            shouldMatchPrefix: false,
```

Comment:
Since this was the only case where we used the prefix match mode, perhaps it's better to remove support for the option entirely...?